python/cuml/common/input_utils.py

```diff
@@ -398,11 +398,21 @@ def input_to_cupy_array(X,
                         check_cols=False,
                         check_rows=False,
                         fail_on_order=False,
-                        force_contiguous=True) -> cuml_array:
+                        force_contiguous=True,
+                        fail_on_nan=True) -> cuml_array:
```

Comment:
We might consider renaming this option to `fail_on_null`. NaNs already work just fine; it's nulls that cause problems.

Reply:
Yes, you're right. Done.